mk/config.mk

```diff
@@ -609,12 +609,18 @@ CFG_CRYPTOLIB_DIR ?= core/lib/libtomcrypt
 # that would set = n.
 $(call force,CFG_CORE_MBEDTLS_MPI,y)
 
+# Defines the number of PKCS11 token implemented by the PKCS11 TA
+CFG_PKCS11_TA_TOKEN_COUNT ?= 3
+
 # Enable PKCS#11 TA's TEE Identity based authentication support
 CFG_PKCS11_TA_AUTH_TEE_IDENTITY ?= y
 
 # Enable PKCS#11 TA's C_DigestKey support
 CFG_PKCS11_TA_ALLOW_DIGEST_KEY ?= y
 
+# Enable PKCS#11 TA's C_DigestKey support
```

Comment:
This comment is weird. So if `CFG_PKCS11_TA_HEAP_SIZE` is not defined or has a different value, C_DigestKey is not enabled?

Reply:
oh sorry, looks I didn't double check this. obviously not the expected description.

Reply:
note we already do the same with `CFG_PKCS11_TA_TOKEN_COUNT` in [_ta/pkcs11/src/pkcs11_token.c_](https://github.com/OP-TEE/optee_os/blob/3.14.0/ta/pkcs11/src/pkcs11_token.c#L28-L32)